src/jrd/jrd.cpp

```diff
@@ -1342,7 +1343,7 @@ static void		start_transaction(thread_db* tdbb, bool transliterate, jrd_tra** tr
 static void		rollback(thread_db*, jrd_tra*, const bool);
 static void		purge_attachment(thread_db* tdbb, StableAttachmentPart* sAtt, unsigned flags = 0);
 static void		getUserInfo(UserId&, const DatabaseOptions&, const char*,
-	const RefPtr<const Config>*, bool, Mapping& mapping, bool);
```

Comment:
Also not related with this PR

Reply:
It is. The check for privilege was moved into createDatabase() to allow impersonalization making the parameter obsolete and unused.